cmake/GenerateLayerJson.cmake

```diff
@@ -10,7 +10,7 @@ function(GENERATE_LAYER_JSON_FILE TARGET RELATIVE_PATH_PREFIX LAYER_BINARY IN_FI
 
     set(CONFIG_DEFINES
         -DINPUT_FILE=${IN_FILE}
-        -DVK_VERSION=${VULKAN_VERSION_MAJOR}.${VULKAN_VERSION_MINOR}.${VULKAN_VERSION_PATCH}
+        -DVK_VERSION=${VK_VERSION_STRING}
```

Comment:
`VK_VERSION_STRING` is an implementation detail of VulkanHeaders. Prefer using `VULKAN_HEADERS_VERSION`

https://cmake.org/cmake/help/latest/variable/PROJECT-NAME_VERSION.html